test/cross-browser-testing/browserstack.karma.config.js

```diff
@@ -28,39 +24,22 @@ if (DEBUG === 'true') {
 }
 
 const customLaunchers = {
-    // These 2 browsers pass because they are capable of running ES6 syntax
-    // Our tests currently have ES6 syntax in them.  They need to be transpiled
-    bs_firefox_mac_100: {
```

Comment:
It's not clear to me why we are removing the Firefox launcher. Are we only concerned about Chrome as an "older" browser?

Reply:
There is another ticket in the initiative to determine what browsers to use - go.mparticle.com/work/SQDSDKS-5827.  FF would definitely make a comeback. 

There wasn't ever really an intention to remove the FIrefox one...I ended up just focusing on Chrome for some reason.